docs/sdk/client-side-sdks/android.md

```diff
@@ -73,6 +74,8 @@ protected void onCreate(Bundle savedInstanceState) {
     
     ...
 }
+    //NOTE: It is not recommended to hardcode SDK keys into your application.
+    //Consider storing keys securely and reading from secure storage.
 ```
 
 ### Notifying when DevCycle features are available
```

Comment:
there's a `client.onInitialized` that should be `dvcClient.onInitialized` in the java snippet below